CHANGELOG.md

```diff
@@ -4,10 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.14.1]
```

Comment:
no need to update version because the icons is not used in AP, only for the sake of ui-lib storybook

Reply:
I want to use the newly added two icons

Reply:
@rayhan-ab  did you mean that the font file here is only use for showing icon at storybook?

and in-order to use the new icon we need to update the font file at AP too?

Reply:
use where?

Reply:
yes

Reply:
only update version if the icon is used in a ui-lib component

Reply:
So what do you think should be done now, discard this modification? @rayhan-ab @aylwin-AB

Reply:
> use where?

for Sections feature to reorder the items in section

Reply:
that means, we do it on AP. anyway we can update here as well to remove config-promoter. when doing on AP, please refer to `How to add new icons to accelicons` doc in confluence